DDCore/src/DetectorLoad.cpp

```diff
@@ -21,6 +21,9 @@
 
 // C/C++ include files
 #include <stdexcept>
+#include <csignal>
```

Comment:
please sort alphabetically with the existing include

Reply:
Done